An err has just occurred in <%= @notice.environment_name %>: <%= raw(@notice.message) %>

This err has occurred <%= pluralize @notice.problem.notices_count, 'time' %>. You should really look into it here:

  <%= app_problem_url(@app, @notice.problem) %>


ERROR MESSAGE:

<%= raw(@notice.message) %>


WHERE:

<%= @notice.where %>

<% @notice.app_backtrace.map {|l| "#{l['file']}:#{l['number']}" }.each do |line| %>
  <%= line %>
<% end %>


URL:

<%= @notice.request['url'] %>


BACKTRACE:

<% @notice.backtrace.map {|l| l ? "#{l['file']}:#{l['number']}" : nil }.compact.each do |line| %>
  <%= line %>
<% end %>

